backend/.gitignore

```diff
@@ -0,0 +1,29 @@
+target/
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+
+### NetBeans ###
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+build/
+!**/src/main/**/build/
+!**/src/test/**/build/
+
+### VS Code ###
+.vscode/
```

Comment:
Consider adding `**/tmp/` to the end so we can create tmp/ folders to use as dumping grounds..

Consider adding `**/application-local.properties`, `**/application-local.yml`, `**/application-local.yaml` to use in place of the `.env` config file.

(or add in a later PR)

Reply:
[Added suggestions above in `.gitignore`.](https://github.com/DTS-STN/canadian-dental-care-plan/compare/e4b6b4e5d79d4f716e14946054039a3e1e8e14d9..57c82567f8066b3daa2195f0ca7ad35ecbf1b414#diff-a99100e55e9705d815385682f3d3178e8a212e73f00ac68787d3fe1c4663afd6)

[Also added a sample `application-local.yml` file](https://github.com/DTS-STN/canadian-dental-care-plan/compare/e4b6b4e5d79d4f716e14946054039a3e1e8e14d9..57c82567f8066b3daa2195f0ca7ad35ecbf1b414#diff-b4ec79303ae4243f108a062b9beb607246c8ebbd3dc4677e0180a316467d1523)